packages/node_modules/@webex/plugin-meetings/test/unit/spec/integration/index.js

```diff
@@ -0,0 +1,260 @@
+    });
+
+    it('user starts own PMR gets event', () => {
+      // TODO
+    });
+  });
+
+  // The event was coming but incomplete
+  // 1) Test user doesnt have locus tag information
+
+  // Alice calls bob and bob rejects it
+  xdescribe('End outgoing Call', () => {
+    after(() => {
+      alice.meeting = null;
+      bob.meeting = null;
+    });
+
+    const aliceMeetingAdded = sinon.spy();
+    const bobMeetingAdded = sinon.spy();
+    it('alice dials bob and bob receives meeting added', () => {
+      bob.spark.meetings.on('meeting:added', bobMeetingAdded);
+      alice.spark.meetings.on('meeting:added', aliceMeetingAdded);
+      return alice.spark.meetings.create(bob.emailAddress)
+        .then((m) => {
+          startMeeting(m);
+          alice.meeting = m;
+          return m.join()
+            .then(() => testUtils.waitForSpy(aliceMeetingAdded, 'meeting:added'))
+            .then((m) => {
+              alice.meeting = m.meeting;
+              console.log('alice dialed');
+            });
+        })
+        .then(() => testUtils.waitForSpy(bobMeetingAdded, 'meeting:added'))
+        .then((m) => {
+          bob.meeting = m.meeting;
+        })
+        .catch((err) => {
+          console.log('ERROR JOIN ', err);
+          throw err;
+        });
+    });
+
+    it('bob and alice have meeting object', () => {
+      //* * acknoledging makes a difference */
+      bob.meeting.acknowledge('INCOMING');
+      assert.equal(bob.meeting.sipUri, alice.emailAddress);
+      assert.equal(alice.meeting.sipUri, bob.emailAddress);
+      assert.equal(bob.meeting.state, 'IDLE');
+      assert.equal(alice.meeting.state, 'JOINED');
+    }).timeout(20000);
+
+    it('alice leaves the meeting', () => alice.meeting.leave()
+      .then(() => testUtils.waitForStateChange(alice.meeting, 'LEFT')));
+
+    it('alice and bob meeting object deleted', () => Promise.all([
+      testUtils.waitForCallEnded(alice, bob.emailAddress),
+      testUtils.waitForCallEnded(bob, alice.emailAddress)
+    ])
+      .then(() => {
+        assert.equal(alice.spark.meetings.getMeetingByType('sipUri', bob.emailAddress), null);
+        assert.equal(bob.spark.meetings.getMeetingByType('sipUri', alice.emailAddress), null);
+      }));
+  });
+
+  // Alice calls bob and bob rejects it
+  describe('reject Incoming Call', () => {
+    const aliceMeetingAdded = sinon.spy();
+    const bobMeetingAdded = sinon.spy();
+    it('alice dials bob and bob receives meeting added', () => {
+      bob.spark.meetings.on('meeting:added', bobMeetingAdded);
+      alice.spark.meetings.on('meeting:added', aliceMeetingAdded);
+      return alice.spark.meetings.create(bob.emailAddress)
+        .then((m) => {
+          startMeeting(m);
+          alice.meeting = m;
+          return m.join()
+            .then(() => testUtils.waitForSpy(aliceMeetingAdded, 'meeting:added'))
+            .then((m) => {
+              alice.meeting = m.meeting;
```

Comment:
Do we need to set this again since ti's already set above?

Reply:
removed it